kaff4-core/kaff4-core-okio/api/kaff4-core-okio.api

```diff
@@ -1,35 +1,7 @@
-public final class net/navatwo/kaff4/io/AutoCloseableCloseOnThrowExtensionsKt {
```

Comment:
This is a breaking change, it moves content from package-to-package, which would break anyone using this library.

... which no one is at this time...